specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/JobAgents.json

```diff
@@ -362,6 +416,24 @@
           }
         }
       }
+    },
+    "JobAgentUserAssignedIdentity": {
```

Comment:
>JobAgentUserAssignedIdentity [](http://example.com/codeflow?start=5&length=28)

Can you use `$ref` to 
UserAssignedIdentity
in
[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json#L50C6-L50C28](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json#L50C6-L50C28)

instead of defining your own user assigned identity?